modules/theme/src/themes/wso2is/assets/images/icons/solid-icons/twinkle-ai-solid.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="14" height="14" viewBox="0 0 439 431" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Let's add the license header.

Is there any specific reason we are not taking this icon from @oxygen-ui/react-icons?

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/5961/commits/891f646b2c63094d4467060253aa82c4bd6751b5